compiler/test/stdlib/array.test.gr

```diff
@@ -94,6 +94,11 @@ Array.forEachi(addToSum, arr1)
 
 assert sum == 8
 
+// Array.reverse
+
+assert Array.reverse([> ]) == [> ] // Empty
+assert Array.reverse([> 1, 2, 3, 4]) == [> 4, 3, 2, 1]
```

Comment:
Based on @ospencer's review, I might suggest an explicit test for this not being in-place; just so we can make sure that behavior doesn't change in the future
```suggestion
let testArr = [> 1, 2, 3, 4]
assert Array.reverse(testArr) == [> 4, 3, 2, 1] // array is reversed properly
assert testArr == [> 1, 2, 3, 4] // original array is not modified
```

Reply:
Good idea.